Blog/gst-build/cross-building-with-gst-build.md

```diff
@@ -104,14 +104,14 @@ After mounting the folder or copying it to your target, you have to set up a few
  * GST_PLUGIN_PATH=$DESTDIR/usr/local/lib/gstreamer-1.0
  * GST_OMX_CONFIG_DIR=$DESTDIR/usr/local/etc/xdg
 
-A python script is also available [here](https://github.com/dabrain34/gstreamer-toolkit/blob/master/gst-build-helper/cross-gst-uninstalled.py) to setup the correct environment for your environment.
+A python script is also available [here](https://github.com/dabrain34/gstreamer-toolkit/blob/master/gst-build-helper/cross-gst-uninstalled.py) to setup the correct environment variables for your target.
 
 
 #### Building a dependency such as kmssink
 
 In order to build a dependency such as `kmssink` which depends on `libdrm`. You'll need to get a proper sysroot with all the libraries, which `kmssink` depends on.
 
-Regarding a root file-system with `libdrm`, I generated one with [cerbero](https://gitlab.freedesktop.org/gstreamer/cerbero) where cross compiling could be described in a next blog post :)
+Regarding a root file-system with `libdrm`, I generated one with [cerbero](https://gitlab.freedesktop.org/gstreamer/cerbero) where cross compiling could be described in a next blog post :) But you should normally have it as part of your sysroot.
```

Comment:
yes this part I might need to remove it seen that we did not really agree on the recipe availabiltiy in cerbero

Reply:
Ok, let me know when you're ready.. Mark keeps on pinging me !